pandas/_libs/lib.pyi

```diff
@@ -30,7 +30,7 @@ from enum import Enum
 class _NoDefault(Enum):
     no_default = ...
 
-no_default: Final = _NoDefault.no_default  # noqa
+no_default: Final = ...
```

Comment:
I think this will break some type annotations

Reply:
Ah okay thanks I'll add back the noqa